include/deal.II/fe/fe_rt_bubbles.h

```diff
@@ -0,0 +1,144 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2018 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii__fe_raviart_thomas_bubbles_h
+#define dealii__fe_raviart_thomas_bubbles_h
+
+#include <deal.II/base/config.h>
+#include <deal.II/base/table.h>
+#include <deal.II/base/polynomials_rt_bubbles.h>
+#include <deal.II/base/polynomial.h>
+#include <deal.II/base/tensor_product_polynomials.h>
+#include <deal.II/base/geometry_info.h>
+#include <deal.II/fe/fe.h>
+#include <deal.II/fe/fe_poly_tensor.h>
+
+#include <vector>
+
+DEAL_II_NAMESPACE_OPEN
+
+/**
+ * This class implements a curl-enhanced Raviart-Thomas elements,
```

Comment:
Is there a paper that describes these elements and that you could reference somewhere in the documentation of this class? I think this would give readers the ability to find more background material.

Reply:
Yes, I completely forgot to reference the paper. It is not published yet, but is available on arXiv, should I put the link to it in the documentation or just the title and authors?

Reply:
Yes, reference the arXiv paper. If the paper is later published, you can always update the link.